src/servers/Scripts/quest/ManSea001.cpp

```diff
@@ -27,102 +32,98 @@ class ManSea001 : public EventScript
 
    void Scene00000( Entity::Player& player )
    {
-      auto callback = [this]( Entity::Player& player, const Event::SceneResult& result )
+      player.playScene( m_id, 0, 8192,
+         [&]( Entity::Player& player, const Event::SceneResult& result )
       {
-         if( result.param2 == 1 )
+         if ( result.param2 == 1 )
          {
             player.setOpeningSequence( 2 );
             Scene00001( player );
          }
-      };
-
-      player.playScene( getId(), 0, HIDE_HOTBAR, 0, 0, callback );
+      } );
    }
 
    void Scene00001( Entity::Player& player )
    {
-      player.playSceneChain( getId(), 1, DISABLE_SKIP | HIDE_HOTBAR | SET_BASE, bindScene( &ManSea001::Scene00002 ) );
+      player.playSceneChain( m_id, 1, DISABLE_SKIP | HIDE_HOTBAR | SET_BASE, bindScene( &ManSea001::Scene00002 ) );
    }
 
    void Scene00002( Entity::Player& player )
    {
-      player.playSceneChain( getId(), 2, NONE, bindScene( &ManSea001::Scene00003 ) );
+      player.updateQuest( m_id, 1 );
+      player.playSceneChain( m_id, 2, NONE, bindScene( &ManSea001::Scene00003 ) );
    }
 
    void Scene00003( Entity::Player& player )
    {
-      auto callback = [this]( Entity::Player& player, const Event::SceneResult& result )
-      {
-         player.playScene( OPENING_EVENT_HANDLER, 0x1E, HIDE_HOTBAR | NO_DEFAULT_CAMERA, 1, 0 );
-      };
-
-      player.playScene( getId(), 3, NONE, 0, 0, callback );
+      player.playScene( OPENING_EVENT_HANDLER, 0x1E, HIDE_HOTBAR | NO_DEFAULT_CAMERA, 1, 0 );
```

Comment:
This is a bit weird, because playscene0003 is supposed to play scene 3, you skip scene 3 and go straight to 0x1E

Reply:
The result is the same cause Scene00003 is an empty scene (according to the LUA file) and OPENING_EVENT_HANDLER works with the playSceneChain from Scene00002

Reply:
If that is the case, do not call it Scene00003 if it really is playing 0x1E ( 30 ) since it really isn't playing 3. Otherwise it gets quite confusing

Reply:
That call Scene00030 from OpeningLimsa (Scene00030 is not present into cpp autogenerated)

Reply:
Fact is still that the call Scene0003 make no sense if it is not actually playing scene 3. It logically is flawed. If all it does is call to the opening handler, then sure ( should be named accordingly ) although i bet retail still uses scene3 as dummy to call to the openinghandler just as in gridania opening.

Reply:
Check on retail:
Scene00002 call Scene00003 -> Scene00003 call opening.Scene00030